src/database/models/reviews.js

```diff
@@ -55,5 +57,45 @@ module.exports = (sequelize, DataTypes) => {
 		}
 	)
 
+	Review.addHook('afterSave', async (instance, options) => {
+		try {
+			//get resource type
+			const resource = await sequelize.models.Resource.findOne(
+				{
+					where: { id: instance.resource_id },
+					attributes: ['id', 'organization_id', 'type'],
+				},
+				{ raw: true }
+			)
+
+			if (resource?.id) {
```

Comment:
@priyanka-TL in which case resource id won't be there

Reply:
@aks30 just to reverify its added, i have removed that